src/django_components/commands/create.py

```diff
@@ -199,16 +199,20 @@ def handle(self, *args: Any, **kwargs: Any) -> None:
 
                     @register("{name}")
                     class {name.capitalize()}(Component):
-                        template_file = "{name}/{template_filename}"
+                        template_file = "{template_filename}"
```

Comment:
Updated the boilerplate that's generated by the command that creates component files.